{{cookiecutter.project_name}}/pyproject.toml

```
@@ -8,6 +8,11 @@ readme = "README.rst"
 homepage = "https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}"
 repository = "https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}"
 documentation = "https://{{cookiecutter.project_name}}.readthedocs.io"
+{% if cookiecutter.package_name != cookiecutter.project_name.replace('-', '_') -%}
+packages = [
+    { include = "{{cookiecutter.package_name}}", from = "src" },
+]
+{%- endif %}
```

> With this version we have a line break between documentation and classifiers keys when no packages key is needed.

Good catch! So the minus in the `endif` should move to the closing tag.

```suggestion
{% endif -%}
```